### ISSUE TITLE
Make the Cobertura profile inactive by default

### ISSUE THREAD

**Issue by lennartj** [mojohaus/mojo-parent#59]
... and consider moving to Clover instead. It works well with Java 8.

**Comment by khmarbaise** [comment]
Also an option would be JaCoCo ?

**Comment by slachiewicz** [comment]
If this plugin is enabled, you can not run site generation under Java 12 (tested with extra-enforcer-rules)

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.7.1:site (default-site) on project extra-enforcer-rules: failed to get report for org.codehaus.mojo:cobertura-maven-plugin: Plugin org.codehaus.mojo:cobertura-maven-plugin:2.7 or one of its dependencies could not be resolved: Could not find artifact com.sun:tools:jar:0 at specified path jdk-11.0.1/../lib/tools.jar -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.7.1:site (default-site) on project extra-enforcer-rules: failed to get report for org.codehaus.mojo:cobertura-maven-plugin
```

**Comment by andham** [comment]
What should we do here? Is JaCoCo they way to go nowadays?

**Comment by andham** [comment]
I've added openjdk11 to the travis ci build and it now shows this problem with cobertura by failing all opendjk 11 builds.

**Comment by andham** [comment]
Fixed by #7, which removes cobertura-m-p all together.

**Comment by mfriedenhagen** [comment]
> Fixed by #7, which removes cobertura-m-p all together.

Should this be #77 :-)

**Comment by andham** [comment]
@mfriedenhagen I think it should be #79, which is the fix (PR). I blame my keyboard for the typo.
